plugins/woocommerce/tests/php/src/Internal/FraudProtection/DecisionHandlerTest.php

```
@@ -66,6 +70,26 @@ public function test_apply_allow_decision(): void {
+	 * causes subsequent fraud checks to return "allow" (due to lower cart value).
+	 */
+	public function test_allow_decision_does_not_overwrite_blocked_session(): void {
+		$this->session_manager
+			->method( 'is_session_blocked' )
+			->willReturn( true );
+
+		$this->session_manager
+			->expects( $this->never() )
+			->method( 'allow_session' );
+
+		$result = $this->sut->apply_decision( ApiClient::DECISION_ALLOW, array( 'session_id' => 'test' ) );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)